docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+| Generate transfer structure report                | No       |
+| Perform file identification (Transfer)            | Yes      |
+| Extract packages                                  | No       |
+| Delete packages after extraction                  | No       |
+| Perform policy checks on originals                | No       |
+| Examine contents                                  | Yes      |
+| Create SIP(s)                                     | Create single SIP and continue processing |
+| Perform file format identification (Ingest)       | No, use existing data |
+| Normalize                                         | Normalize for preservation and access |
+| Approve normalization                             | Yes      |
+| Generate thumbnails                               | No       |
+| Perform policy checks on preservation derivatives | No       |
+| Perform policy checks on access derivatives       | No       |
+| Bind PIDs                                         | No       |
+| Document empty directories                        | No       |
+| Reminder: add metadata if desired                 | Continue |
+| Transcribe files (OCR)                            | No       |
+| Perform file format identification command (Submission documentation & metadata) | No |
+| Select compression algorithm                      | Gzipped tar |
+| Select compression level                          | 5 - normal compression mode |
+| Store AIP                                         | Yes      |
+| Store AIP location                                | Wellcome AIP Storage |
+| Upload DIP                                        | Do not upload |
+| Store DIP                                         | Do not store |
+| Store DIP location                                | Default location |
+
+### Storage
+
+The final stage of the workflow involves the creation of Archive Information Packages (AIPs) and safely moving them to long term storage. The parts of the process are handled automatically by Archivematica and the new storage service.
+
+Once an AIP has been successfully stored in the storage service,  its unique identifier should be added to the equivalent item record in Calm, so that it can be easily located using the catalogue. For now this addition needs to be a manual process, but we hope to be able to automate it in the future.
+
+| Step                | Description |
+|---------------------|-------------|
+| Create SIP          | Archivematica automatically creates a Submission Information Package (SIP), that describes the files that will be stored as an Archive Information Package (AIP). |
+| Store AIP           | Archivematica automatically creates an AIP from the SIP and ensures it is correctly stored in the storage service. |
+| Link from catalogue | Add the unique identifier of the AIP to the associated item record in Calm, so that the catalogue can be used to find files in the storage service. |
```

Comment:
How will this link work? Click on it and it takes a user into the storage service and depending on their access they can view/download the images as necessary?

Reply:
There's a whole missing section on DIPs here really. I would like to get this written up sooner rather than later, even if for practical implementation the priority is getting ingest into storage configured. Not considering access requirements has been a big barrier to progressing born digital in the past and I'd like to avoid that happening again.

Reply:
This is one area where we may diverge from Archivematica slightly. It can create a separate DIP, but it's designed for uploading to Atom. I think we can do everything we need to do better with a single package, the AIP, which contains the originals and any copies created for preservation or access.

There is a real limit to the degree that we can consider access up front. Making this content available is not currently a priority for the next year at least.

We should definitely cover off what gets put in to Calm and where during the more detailed conversations around cataloguing implications.